tiled/server/authentication.py

```diff
@@ -1159,8 +1190,13 @@ async def revoke_session_by_id(
         return Response(status_code=HTTP_204_NO_CONTENT)
 
     async def slide_session(refresh_token, settings, db):
+        if isinstance(settings.authenticator, ProxiedOIDCAuthenticator):
+            return
```

Comment:
Should: Add a comment about refreshing the token being the responsibility of the client/proxy in this case